GVFS/GVFS.Common/FileSystem/HooksInstaller.cs

```diff
@@ -13,6 +13,12 @@ namespace GVFS.Common.FileSystem
     public static class HooksInstaller
     {
         private static readonly string ExecutingDirectory;
+        private static readonly HookData[] Hooks = new[]

```

Comment:
Super minor: Maybe rename to `NativeHookData[] NativeHooks` (since the pre\post command hooks are not part of this collection)?